src/gui/editors/SongEditor.cpp

```diff
@@ -124,6 +120,17 @@ QColor positionLine::lineColor() const
 void positionLine::setLineColor( const QColor & c )
 { m_lineColor = c; }
 
+// NOTE: the move() implementation fixes a bug where the position line would appear
+// in an unexpected location when positioned at the start of the track
+void positionLine::zoomChange( double zoom )
+{
+	move( x() + width() - 1, y() );
+	resize( 8.0 * zoom, height() );
+	move( x() - width() + 1, y() );
```

Comment:
Actually, I think I could change this up a little by using a temporary variable. I think that would make it quicker too without actually moving anything.

Reply:
I'll commit by today evening with a fix.

Reply:
Done.